pac4j-cas/src/test/java/org/pac4j/cas/client/CasClientTests.java

```diff
@@ -127,7 +127,7 @@ public void testBackLogout() {
         final MockWebContext context = MockWebContext.create().addRequestParameter(CasConfiguration.LOGOUT_REQUEST_PARAMETER, LOGOUT_MESSAGE)
             .setRequestMethod("POST");
         TestsHelper.expectException(() -> casClient.getCredentials(context), HttpAction.class, "back logout request: no credential returned");
-        assertEquals(200, context.getResponseStatus());
+        assertEquals(204, context.getResponseStatus());
```

Comment:
204 is for Ok without content. The proposed change was for the `ApplicationLogoutLogic`, but the code is used for CAS logout / proxy as well. I guess it's safer for now to keep 200 for CAS while updating the CAS server itself.